Implement site search

alexcarol opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #24 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/suche?q=traktor
- After: https://issue-24--eder-group--hlxsites.hlx.page/suche?q=traktor

Notes:
- Search algorithm is not identical, it currently matches if any of the search terms appear in either the title, description or page content. If we want to refine it we're going to need a specification of what's wanted.
- Sidebar is not implemented (as discussed previously)
- Pagination format is reused from news, which means that it's slightly different from preexisting eder search. Whichever one we choose we should have it be consistent amongst both pages.
- Pagination css should probably be unified, what could be a good css file for that?

mhaack commented:
Nice. Found some minor style issues:
* keyword highlight in the search result list should be bold
* input field and button should not have round corners
* lets make input filed and button the same high
* since we have more space we could give the input field little more width as well

alexcarol commented:
> Nice. Found some minor style issues:
> 
> * keyword highlight in the search result list should be bold
> * input field and button should not have round corners
> * lets make input filed and button the same high
> * since we have more space we could give the input field little more width as well

Addressed PR issues, let me know if you need me to adjust anything.
